stdlib/2/abc.pyi

```diff
@@ -12,7 +12,7 @@ def abstractmethod(funcobj: Any) -> Any: ...
 class ABCMeta(type):
     # TODO: FrozenSet
     __abstractmethods__ = ...  # type: Set[Any]
-    __doc__ = ...  # type: str
+    __doc__ = ...  # type: Optional[str]
```

Comment:
No class other than object actually needs to declare `__doc__` at all. They just inherit it from object. I think the stubs have it because the stub generator puts it in.